config/initializers/airbrake.rb

```diff
@@ -1,7 +1,6 @@
-# This file is overwritten on deploy
-#
 Airbrake.configure do |config|
-  # Adding "production" to the development environments causes Airbrake not
-  # to attempt to send notifications.
-  config.development_environments << "production"
+  config.api_key = ENV['ERRBIT_API_KEY']
+  config.host = "errbit.#{ENV['GOVUK_APP_DOMAIN']}"
```

Comment:
Yes? You might be missing context, does phrasing it like this make more sense?

It would be nice if this was something like this such that it is fully configurable:
```ruby
  config.host = ENV['ERRBIT_HOST'] or "errbit.#{ENV['GOVUK_APP_DOMAIN']}"
```